packages/asset-swapper/src/quote_consumers/plp_registry.ts

```diff
@@ -0,0 +1,77 @@
+import { ContractFunctionObj } from '@0x/contract-wrappers';
```

Comment:
This should live in `utils` rather than `quote_consumers`

Reply:
I moved the file to `utils`. Thank you for the suggestion